cypress/integration/common/dashboard_sample_data_spec.js

```diff
@@ -247,6 +247,7 @@ export function dashboardSanityTests() {
 
       describe('checking discover', () => {
         before(() => {
+          cy.setAdvancedSetting({ 'discover:v2': false });
```

Comment:
@leanneeliatra I noticed that you made this change, just want to confirm is this by intention? As this is unrelated to security dashboards plugin.

Reply:
Yes, thanks for spotting that ruanyl. I have removed it.